### PR TITLE
Created Post Activity for posting and added new svg icons to the draw…

### PR DESCRIPTION
**Pull request by latnok25**
…able folder also made layouts for the bottom sheet dialog for post visibbility and offer/request

**What this PR does**: Added Activity For Create_Post UI also added layouts for BottomSheetDialogs

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for reviewers**:

**Additional comments**:

### CODE REVIEW

**Review by stavares843 — COMMENTED** [review]
is this visible on any flow atm? 

tried a couple flows, didn't appear 

can you add a screenshot?

also, there's a couple comments from the bot

**Comment by latnok25** [comment]
> is this visible on any flow atm?
> 
> tried a couple flows, didn't appear
> 
> can you add a screenshot?
> 
> also, there's a couple comments from the bot

Screen Shot 2020-07-03 at 5 06 14 PM
Screen Shot 2020-07-03 at 5 14 44 PM
Screen Shot 2020-07-03 at 5 17 30 PM

I have resolved those comments

**Review by stavares843 — COMMENTED** [review]
- on design the "are you offering or request help?" and the "who can see this post?" are a modal instead of a new view
- overall design seems to have a different font 
- on "create a post" view is missing the copy on the button on the bottom

**Comment by latnok25** [comment]
> * on design the "are you offering or request help?" and the "who can see this post?" are a modal instead of a new view
> * overall design seems to have a different font
> * on "create a post" view is missing the copy on the button on the bottom

The View was created as a custom view to be used by the model which is going to be bottom_dialog_sheet 

I used the same fonts from zeplin Poppins-Bold for Bold fonts and WorkSans-Regular for regular fonts or light fonts

I don't really get what you mean buy copy but if its the missing text in the button, The button is currently disabled that's why the text is missing

**Comment by stavares843** [comment]
yes but when the button is deactivated, the copy should appear 


design

Screenshot 2020-07-03 at 19 56 19

**Comment by latnok25** [comment]
> yes but when the button is deactivated, the copy should appear
> 
> design
> 
> Screenshot 2020-07-03 at 19 56 19

Screen Shot 2020-07-03 at 8 14 11 PM

made a quick fix

File: app/src/main/java/com/fightpandemics/Create_Post/Create_Post.kt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package com.fightpandemics.Create_Post
```
Comment (rajasgandhi) [inline app/src/main/java/com/fightpandemics/Create_Post/Create_Post.kt:1]: Looks good but I would remove the underscore here

**Review by rajasgandhi — APPROVED** [review]

File: app/src/main/java/com/fightpandemics/Create_Post/Create_Post.kt (hunk 1 above)
Comment (stavares843) [reply app/src/main/java/com/fightpandemics/Create_Post/Create_Post.kt:1]: agree

**Pull request closed without merging**